<?xml version="1.0" ?>
<sdf version="1.6">
  <model name="s2l2_kit_tray">
    <static>true</static>
    <allow_auto_disable>false</allow_auto_disable>

    <link name="base_link">
      <inertial auto="true"/>
      
      <visual name="visual">
        <geometry>
          <mesh>
            <uri>model://demo_parts/s2l2_kit_tray/meshes/s2l2_kit_tray.stl</uri>
          </mesh>
        </geometry>
        <material>
          <ambient>1.0 1.0 1.0 1.0</ambient>
          <diffuse>1.0 1.0 1.0 1.0</diffuse>
        </material>
      </visual>

      <collision name="collision">
        <pose>0 0 0 0 0 0</pose>
        <geometry>
          <mesh>
            <uri>model://demo_parts/s2l2_kit_tray/meshes/s2l2_kit_tray.stl</uri>
          </mesh>
        </geometry>
        <max_contacts>5</max_contacts>
      </collision>
    </link>

    <link name="lg_1">
      <inertial auto="true"/>
      <collision name="collision">
        <density>1</density>
        <pose>-0.052 0.06 -0.012 0 0 0</pose>
        <geometry>
          <box>
            <size>0.02 0.02 0.0001</size>
          </box>
        </geometry>
      </collision>
    </link>

    <link name="lg_2">
      <inertial auto="true"/>
      <collision name="collision">
        <density>1</density>
        <pose>0.052 0.06 -0.012 0 0 0</pose>
        <geometry>
          <box>
            <size>0.02 0.02 0.0001</size>
          </box>
        </geometry>
      </collision>
    </link>

    <link name="sg_1">
      <inertial auto="true"/>
      <collision name="collision">
        <density>1</density>
        <pose>-0.045 -0.045 -0.012 0 0 0</pose>
        <geometry>
          <box>
            <size>0.02 0.02 0.0001</size>
          </box>
        </geometry>
      </collision>
    </link>

    <link name="sg_2">
      <inertial auto="true"/>
      <collision name="collision">
        <density>1</density>
        <pose>0.045 -0.045 -0.012 0 0 0</pose>
        <geometry>
          <box>
            <size>0.02 0.02 0.0001</size>
          </box>
        </geometry>
      </collision>
    </link>

    <joint name="lg_1_joint" type="fixed">
      <parent>base_link</parent>
      <child>lg_1</child>
    </joint>

    <joint name="lg_2_joint" type="fixed">
      <parent>base_link</parent>
      <child>lg_2</child>
    </joint>

    <joint name="sg_1_joint" type="fixed">
      <parent>base_link</parent>
      <child>sg_1</child>
    </joint>
    <joint name="sg_2_joint" type="fixed">
      <parent>base_link</parent>
      <child>sg_2</child>
    </joint>

  </model>
</sdf>